data/trie/patriciaMerkleTrie.go

```diff
@@ -316,3 +365,26 @@ func emptyTrie(root []byte) bool {
 	}
 	return false
 }
+
+// Rollback invalidates the hashes that correspond to the given root hash from the eviction waiting list
+func (tr *patriciaMerkleTrie) Rollback(rootHash []byte) error {
+	_, err := tr.dbEvictionWaitingList.Evict(rootHash)
```

Comment:
I think you can log the evicted hash. Also in the Prune() func

Reply:
This function returns a slice of hashes. In a real life situation, this will return thousands of hashes. I don't think it will help in any way to log all those hashes.